code/game/objects/items/mountable_frames/trophy_mount.dm

```diff
@@ -0,0 +1,153 @@
+/obj/item/mounted/frame/trophy_mount
+	name = "trophy mount"
+	desc = "A wooden trophy mount."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "trophy_mount"
+	w_class = W_CLASS_MEDIUM
+	flags = FPRINT
+	w_type = RECYK_WOOD
+	frame_material = /obj/item/stack/sheet/wood
+	sheets_refunded = 2
+	autoignition_temperature = AUTOIGNITION_WOOD
+	var/obj/item/held_item
+	var/list/params_list
+	var/icon/clicked //Because BYOND can't give us runtime icon access, this is basically just a click catcher
```

Comment:
How is this a click catcher? All this is used for is getting the icon size.

Reply:
I didn't write the comment, I copied it from table code. I can remove it if you think it's inaccurate.

Reply:
Yes then remove it.